sdk/nodejs/output.ts

```diff
@@ -534,6 +534,20 @@ export function secret<T>(val: Input<T | undefined>): Output<Unwrap<T | undefine
         o.isKnown, Promise.resolve(true), o.allResources!());
 }
 
+/**
+ * [unsecret] behaves the same as [output] except the returned output takes the existing output and unwraps the secret
+ */
+export function unsecret<T>(val: Output<T>): Output<T> {
```

Comment:
I could go either way, but I am wondering if we should consider making this accept `Input<T>` like the existing `output` and `secret` functions (in the implementation, if `val` isn't an `Output`, then it'd be a no-op). @lukehoban, do you have an opinion?